packages/foam-vscode/static/dataviz/index.html

```diff
@@ -2,14 +2,10 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <script data-replace src="./d3.v6.min.js"></script>
-    <script data-replace src="./force-graph.1.40.5.min.js"></script>
-    <script data-replace src="./dat.gui.min.js"></script>
-    <style>
-      body {
-        overflow: hidden;
-      }
-    </style>
+    <script replace-src="./d3.v6.min.js"></script>
```

Comment:
what's the reason for using `replace-src`?

the reason why we use `data-replace src=` is that it's a valid html tag, which means you can still open the webpage in a browser (outside of vscode) if you want to do some testing (the dev cycle tends to be faster with that approach, although I have never documented it)

Reply:
Nothing in particular, I needed to generalize the mechanism to also include href and I just did it this way, but I see your point.